src/overlay_ordering.cpp

```diff
@@ -0,0 +1,25 @@
+#include "overlay_ordering.h"
+
+std::unordered_map<std::string, int> base_mutation_overlay_ordering;
+std::unordered_map<std::string, int> tileset_mutation_overlay_ordering;
```

Comment:
More generally, I'd say default to map unless you have a strong indication
that it is going to matter.